client/jetpack-connect/store-footer.tsx

```diff
@@ -6,12 +6,16 @@ import React from 'react';
 /**
  * Internal dependencies
  */
+import { getJetpackCROActiveVersion } from 'calypso/my-sites/plans-v2/abtest';
 import JetpackFAQ from 'calypso/my-sites/plans-features-main/jetpack-faq';
+import JetpackFAQi5 from 'calypso/my-sites/plans-features-main/jetpack-faq-i5';
 
 export default function StoreFooter() {
+	const JetpackFAQComponent = getJetpackCROActiveVersion() === 'i5' ? JetpackFAQi5 : JetpackFAQ;
```

Comment:
This is not ideal, will try to deal with this as we know better what we're doing with all these iterations soon.